haiku-apps/bephonebook/bephonebook-1.2.recipe

```diff
@@ -0,0 +1,47 @@
+SUMMARY="A phone book manager"
+DESCRIPTION="BePhoneBook is a colorful phonebook made to help you stay \
+organized with your family's or friends' phone numbers."
+HOMEPAGE="https://github.com/HaikuArchives/BePhoneBook"
+COPYRIGHT="1999 Michael Melloff \
+	Fabien Fulhaber"
+LICENSE="libsanta"
```

Comment:
```suggestion
LICENSE="libsanta
	Unknown"
```

As [mentioned](https://github.com/haikuports/haikuports/pull/3354#discussion_r232650490), the main files don't have any license associated with, so we'd still need `Unknown` here.